planetarium/NineChronicles
# Add properties to `RecipeViewData`.

### Description

1. Add properties on `RecipeViewData`.
2. Extend `RecipeView` to apply newly added properties.

### How to test

1. Go to workshop.
2. Check if `RecipeView` is properly implemented to reflect newly added properties.

### Related Links
https://app.asana.com/0/958521740385861/1201705327357522

### Required Reviewers
@planetarium/9c-dev @BBeom06 

### Screenshot
![image](https://user-images.githubusercontent.com/26648527/152335963-389d38c7-d022-409e-8ea0-f53b4236ae15.png)
